src/Twig/Handler/ImageHandler.php

```diff
@@ -29,16 +29,16 @@ public function __construct(Silex\Application $app)
     /**
      * Helper function to make a path to an image.
      *
-     * @param string         $filename Target filename
-     * @param string|integer $width    Target width
-     * @param string|integer $height   Target height
-     * @param string         $crop     String identifier for cropped images
+     * @param string  $filename Target filename
+     * @param integer $width    Target width
+     * @param integer $height   Target height
+     * @param string  $crop     String identifier for cropped images
      *
      * @return string Image path
      */
-    public function image($filename, $width = '', $height = '', $crop = '')
+    public function image($filename = '', $width = 0, $height = 0, $crop = '')
```

Comment:
If you're tackling stuff like this in v3/master, a note that these really should default to `null` and checked accordingly. We've spent a lot of effort removing this behaviour.

While in this case it is highly unlikely that someone would want a file name that is `''` or  width/height that is `0`, it is a slippery slope of assumption… and we know what Samuel L. Jackson says about assumption. That said, there are time this is valid, but I don't think this is one of those.